src/main/java/ladder/domain/LadderGame.java

```diff
@@ -0,0 +1,29 @@
+package ladder.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LadderGame {
+    private final Players players;
+    private final Prizes prizes;
+    private final Ladder ladder;
+
+    public LadderGame(Players players, Prizes prizes, Ladder ladder) {
+        this.players = players;
+        this.prizes = prizes;
+        this.ladder = ladder;
+    }
+
+    public LadderGameResult playGame() {
+        Map<Player, Prize> gameResultMap = new HashMap<>();
+
+        for (int position = 0; position < players.count(); position++) {
+            Player player = players.getPlayerByPosition(position);
+            Prize prize = prizes.getPrizeByPosition(ladder.move(position));
+
+            gameResultMap.put(player, prize);
+        }
```

Comment:
```suggestion
players.stream()
    .collect(Collectors.toMap(
        player -> player,
        player -> prizes.getPrizeByPosition(ladder.move(player.position()))
    ));
```

이런식으로 stream으로 작성하는 방식도 고려해보시면 좋을거 같아요!